### ISSUE TITLE
New project: The "English" editor for GitHub

### ISSUE THREAD

**Issue by scripting** [scripting/Scripting-News#48]
From today's Scripting News: 

> Speaking of convolution. I'm trying out the idea of a wizzy editor for GitHub readmes. You get a simple editor, based on Medium-Editor, so you get simple styling, lists, heads. And I found a plug-in that generates Markdown code. So what I push up to GitHub is Markdown, which is the only style that GH renders as HTML as far as I know. It works. Now the question is how to save the text locally between edits. Ideally it would be in Markdown, but the plug-in can't turn Markdown into HTML to initialize the editor. There was a thread on the repo for the project where people asked for this but it appears not to have happened. I tried the hack. Use another MD processor to do the initial conversion, and go from there. Hey Markdown is enough of a standard that it works. What a hack. I love it. PS: I started a thread on my repo for this project if you have questions or opinions. 

This is the thread. Ask or suggest away. ;-)

**Comment by scripting** [comment]
I'm using the editor to work on a new Readme for this repo.

https://github.com/scripting/Scripting-News/blob/master/READMEx.md

Dave

**Comment by scripting** [comment]
Here's a screen shot of the editor as it exists today.

![image](https://user-images.githubusercontent.com/1686843/33621338-610f0a5a-d9b8-11e7-965e-767ab406ee58.png)

**Comment by tedbergeron** [comment]
I'm very interested in this. So is the editor running on the Scripting News website? Then when saving it pushes the changes back to your GitHub repo?

**Comment by scripting** [comment]
Not exactly. In the Settings dialog you can change the repo and path of the file you want to save to. So I could have written a readme for any repo I have editing access to.

**Comment by scripting** [comment]
Here's the howto --

http://this.how/english/

Dave

**Comment by tedbergeron** [comment]
I love it. I just tested it on my repo and it worked: https://github.com/tedbergeron/EnglishEditor/blob/master/message.md

It also worked to change the extention to .adoc and renders well
https://github.com/tedbergeron/EnglishEditor/blob/master/testAsciiDoc.adoc

This is great. Thank you Dave.
